src/configurations.json

```diff
@@ -33,7 +33,7 @@
   "instaLink": "https://www.instagram.com/",
   "instaUsername": "hashir.shoaeb",
   "instaQuerry": "/?__a=1",
-  "aboutDev": "Hashir is a final year Computer Engineering student at NUST CEME. He is vice president student body at Emerging Technologies Research Lab(ETL) and former Intern at RiseTech. He has keen interest in Robotics, and recently working on his Final Year Project. He also has three years' experience in Web Development, working in cross-functional teams and using multiple web platforms. Hashir is passionate about teaching machines how to think.",
+  "aboutDev": "A final year Computer Science Engineer from NUST CEME, Pakistan with 3+ years experience in web development and passionate about teaching machines to think. I’m currently the Vice President of student body at Emerging Technologies Research Lab(ETL) and a former Intern at RiseTech.",
```

Comment:
#1 Can you please change (Computer Science Engineer) to (Computer Engineer)

Reply:
@hashirshoaeb Done.